packaging/installer/UPDATE.md

```diff
@@ -11,7 +11,7 @@ The update procedure depends on how you installed it:
 
 ### Manual update to get the latest git commit
 
-netdata versions older than `v1.12.0-rc2-52` had a `netdata-updater.sh` script in the root directory of the source code, which has now been deprecated. The manual process that works for all versions to get the latest commit in git is to use the `netdata-installer.sh`. The installer preserves your configuration. You just need to be mindful of any installer options you may have used to customize your original installation. 
+netdata versions older than `v1.12.0-rc2-52` had a `netdata-updater.sh` script in the root directory of the source code, which has now been deprecated. The manual process that works for all versions to get the latest commit in git is to use the `netdata-installer.sh`. The installer preserves your custom configuration and updates the the information of the installation in the `.environment` file under the user configuration directory. You just need to be mindful of any installer options you may have used to customize your original installation. 
```

Comment:
> You just need to be mindful of any installer options you may have used to customize your original installation.

"just" :rofl: It is like asking someone to remember how he was installing something couple months ago. Especially when it is not important to "remember" since previous installations were storing those information in `packaging/installer/environment.sh` or `packaging/environment.sh`

Reply:
The settings are no longer stored there if we install from source. If we don't change that, there's nothing to write that can apply to all cases. Let me know if you have something better to suggest.

Reply:
I already suggested. If someone has old installation, then this file exists on his host.

Reply:
I just removed that sentence. It's too confusing to talk about the older and the newer versions with environment.sh vs .environment. Can you make the installer read `/etc/netdata/.enviroment` for the options in a separate PR?

Reply:
No. I cannot.